src/cli/build.rs

```diff
@@ -41,6 +41,10 @@ pub struct Args {
     /// Whether to build incrementally if possible
     #[clap(long, short)]
     pub no_incremental: bool,
+
+    /// The cached directory to use for incremental builds
+    #[clap(long, short)]
+    pub cache_dir: Option<PathBuf>,
```

Comment:
Maybe we can rename this to build_dir? I think cache_dir is a bit confusing given that it doesnt modify the PIXI_CACHE_DIR directory.